src/utils/vacation-days-utils.tsx

```diff
@@ -0,0 +1,43 @@
+import { Typography, Box} from "@mui/material";
+import { theme } from "../theme";
+import { Person } from "../generated/client";
+import strings from "../localization/strings";
+import { useState } from "react";
+import { personsAtom } from "../atoms/person";
+import { useAtom } from "jotai";
+
+/**
+ * Display persons vacation days
+ * @param Person timebank person
+ */
+export const renderVacationDays = (person: Person) => {
```

Comment:
I think it makes more sense for this to be a component perhaps under components/ generics or a new folder componenets/vacations as it is actually is rendering something. Utils are utilities that usually deal with logic rather than rendering something.

Reply:
Moved to components/generics and renamed to vacation-days.tsx